enterprise/frontend/src/metabase-enterprise/moderation/components/ModerationReviewBanner/ModerationReviewBanner.tsx

```diff
@@ -67,4 +62,13 @@ export function ModerationReviewBanner({
       </TextContainer>
     </Container>
   );
-}
+};
+
+// eslint-disable-next-line import/no-default-export -- deprecated usage
+export default _.compose(
+  Users.load({
+    id: (_state: any, props: any) => props.moderationReview.moderator_id,
+    loadingAndErrorWrapper: false,
+  }),
+  connect(mapStateToProps),
+)(ModerationReviewBanner);
```

Comment:
often when i do these little refactors, i try to use named exports to get rid of these little ignore comments. There are probably only a couple imports, right?